dash/include/dash/GlobMem.h

```diff
@@ -105,7 +105,7 @@ class GlobMem
    *       For example, \c dash::LocalAllocator is used in \c dash::Shared
    *       and only called at owner unit.
    */
-  inline GlobMem(
+  explicit GlobMem(
```

Comment:
Nope, `explicit` has nothing to do with `constexpr`. It just prevents implicit conversion from `size_type` to `GlobMem`.